{{cookiecutter.project_dirname}}/scripts/ci_sentry.sh

```diff
@@ -2,26 +2,27 @@
 
 apk update && apk add git
 
-git config --global --add safe.directory ${PROJECT_DIR}
+git config --global --add safe.directory "${PROJECT_DIR}"
 
 if [ "${VAULT_ADDR}" != "" ]; then
   apk add curl jq
 
-  export VAULT_TOKEN=`curl --silent --request POST --data "role=${VAULT_ROLE}" --data "jwt=${CI_JOB_JWT_V2}" ${VAULT_ADDR%/}/v1/auth/gitlab-jwt/login | jq -r .auth.client_token`
+  vault_token=$(curl --silent --request POST --data "role=${VAULT_ROLE}" --data "jwt=${CI_JOB_JWT_V2}" "${VAULT_ADDR%/}"/v1/auth/gitlab-jwt/login | jq -r .auth.client_token)
 
-  export SENTRY_AUTH_TOKEN=`curl --silent --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR%/}/v1/${PROJECT_SLUG}/envs/${ENV_NAME}/sentry | jq -r .data.sentry_auth_token`
-
-  export SENTRY_DSN=`curl --silent --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR%/}/v1/${PROJECT_SLUG}/envs/${ENV_NAME}/${SERVICE_SLUG}/sentry | jq -r .data.sentry_dsn`
+  SENTRY_AUTH_TOKEN=$(curl --silent --header "X-Vault-Token: ${vault_token}" "${VAULT_ADDR%/}"/v1/"${PROJECT_SLUG}"/envs/"${ENV_NAME}"/sentry | jq -r .data.sentry_auth_token)
+  SENTRY_DSN=$(curl --silent --header "X-Vault-Token: ${vault_token}" "${VAULT_ADDR%/}"/v1/"${PROJECT_SLUG}"/envs/"${ENV_NAME}"/"${SERVICE_SLUG}"/sentry | jq -r .data.sentry_dsn)
+  export SENTRY_AUTH_TOKEN
```

Comment:
consider to add an `unset` for sensitive environment variables at point where they become no longer necessary

Reply:
GitLab runner deals with this already, there are a lot of sensitive env vars involved in a single run of a CI job (e.g. the JWT token itself), and they are all handled correctly by the runner. I would not worry about this or anything of the sort.